src/js/routes/result/Result.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import BackButton from '../../components/BackButton.jsx';
+
+const Result = (props) => (
```

Comment:
No worries, it can go. Just safeguarding the table design to keep it simple and not include _everything_ at once. For instance, using an expand/collapse construct could get unwieldy, and give the user no navigational reference.

From the wiki:
`Main page Has:
search field
list of results
clicking on a result row » either expand to show other fields or goto a new page`

To get the ball rolling, I can remove the "Result" pages.